src/socket.io/categories.js

```diff
@@ -206,7 +205,7 @@ function ignoreOrWatch(fn, socket, cid, callback) {
 			}, next);
 		},
 		function (next) {
-			topics.pushUnreadCount(socket.uid, next);
+			topics.pushUnreadCount(socket.uid, next(null, cids));
```

Comment:
please change this line so we dont pass undefined to pushUnreadCount, you can send cids back by adding a new function below